Dockerfile

```diff
@@ -1,18 +1,46 @@
-# syntax=docker.io/docker/dockerfile:1.4
+# syntax=docker.io/docker/dockerfile:1
+
+# This enforces that the packages downloaded from the repositories are the same
+# for the defined date, no matter when the image is built.
+ARG NOBLE_DATE=20240801
+ARG APT_UPDATE_SNAPSHOT=${NOBLE_DATE}T030400Z
```

Comment:
Is there a good reason to choose 3h04 UTC?

Reply:
Just got the value from the example from the site, but I think it could be any value.